src/expr/src/scalar.proto

```diff
@@ -81,7 +81,7 @@ message ProtoUnmaterializableFunc {
         google.protobuf.Empty current_schemas_without_system = 3;
         google.protobuf.Empty current_timestamp = 4;
         google.protobuf.Empty current_user = 5;
-        google.protobuf.Empty mz_cluster_id = 6;
```

Comment:
Should we reserve this one, or are we confident this is never serialized as JSON?

Reply:
I'm 98% confident, but it's a good point. Should probably `reserved` this one to build the good habit.